src/einsteinpy/symbolic/riemann.py

```diff
@@ -55,7 +55,10 @@ def __init__(
     @classmethod
     def from_christoffels(cls, chris, parent_metric=None):
         """
-        Get Riemann Tensor calculated from a Christoffel Symbols
+        Get Riemann Tensor calculated from a Christoffel Symbols Reimann Tensor is given as:
+        .. math::
+           R^{t}{}_{s r n}=\\Gamma^{t}{}_{s n, r} - \\Gamma^{t }{}_{s r, n }
```

Comment:
If the doctests are failing, try adding a blank line at L60 (Though, it should not have an effect ideally). Also, you can try testing the output of this RST directive (`.. math::`...) at https://livesphinx.herokuapp.com/ . 

As an aside, EinsteinPy uses [`numpydocs` guidelines](https://numpydoc.readthedocs.io/en/latest/example.html) for docstrings. You may check the math formatting (and other doc guidelines) there.